spark/src/main/scala/org/apache/spark/sql/delta/SnapshotManagement.scala

```diff
@@ -58,6 +59,22 @@ trait SnapshotManagement { self: DeltaLog =>
 
   @volatile protected var currentSnapshot: CapturedSnapshot = getSnapshotAtInit
 
+  /** Use ReentrantLock to allow us to call `withSnapshotLockInterruptibly` */
+  protected val snapshotLock = new ReentrantLock()
+
+  /**
+   * Run `body` inside `snapshotLock` lock using `withSnapshotLockInterruptibly` so that the thread
```

Comment:
These two comments should probably still say `lockInterruptibly`, no?

Reply:
fixed.